tests/contrib/openstack/test_deferred_events.py

```diff
@@ -304,10 +304,13 @@ def test_get_service_start_time(self, check_output):
     def test_check_restart_timestamps(self, get_service_start_time, log,
                                       clear_deferred_restarts,
                                       get_deferred_restarts):
+        request_time = '2021-02-02 10:19:55'
+        timestamp = datetime.datetime.strptime(
+            'Tue ' + request_time + ' UTC',
+            '%a %Y-%m-%d %H:%M:%S %Z').timestamp()
```

Comment:
the timestamp is seconds since epoch -- but it's timezone specific.  This change was so this test would run successfully on all machines regardless of the locale's timezone. (ie -- it failed on my machine but was okay in other places where there was no TZ set)